docs/yaml/objects/compiler.yaml

```diff
@@ -586,3 +586,68 @@ methods:
     gcc or msvc, but use the same argument syntax as one of those two compilers
     such as clang or icc, especially when they use different syntax on different
     operating systems.
+
+- name: symbol_visibility_header
+  returns: File
+  since: 0.63.0
+  posargs:
+    name:
+      type: str
+      description: Header file name.
+  kwargs:
+    api_variable:
+      type: str
+      description: |
+        Name used for public APIs. Defaults to `MESON_API`.
+    compilation_variable:
+      type: str
+      description: |
+        Name defined only when compiling the library. Defaults to `MESON_COMPILATION`.
+    static_compilation_variable:
+      type: str
+      description: |
+        Name defined only when compiling static library. Defaults to `MESON_STATIC_COMPILATION`.
+  description: |
+    Generate a header file that define a macro to be used to mark all public APIs
+    of a library. Depending on the type of compiler and the platform, this will
+    typically use `__declspec(dllexport)`, `__declspec(dllimport)` or
+    `__attribute__((visibility("default")))`. It is only supported for C and C++
+    compilers.
+
+    If this header is installed it should be included at the beginning of every
+    public headers. If it is not being installed, it should be included at the
+    beginning of every source files, and it is the responsability of every
```

Comment:
After "every" you need to use singular rather than plural case. So "every public header", "every source file" and "every application".

Reply:
Thanks, fixed.